sdk/sphere/Azure.ResourceManager.Sphere/api/Azure.ResourceManager.Sphere.netstandard2.0.cs

```diff
@@ -0,0 +1,611 @@
+    public readonly partial struct CertificateStatus : System.IEquatable<Azure.ResourceManager.Sphere.Models.CertificateStatus>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public CertificateStatus(string value) { throw null; }
+        public static Azure.ResourceManager.Sphere.Models.CertificateStatus Active { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.CertificateStatus Expired { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.CertificateStatus Inactive { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.CertificateStatus Revoked { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Sphere.Models.CertificateStatus other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Sphere.Models.CertificateStatus left, Azure.ResourceManager.Sphere.Models.CertificateStatus right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Sphere.Models.CertificateStatus (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Sphere.Models.CertificateStatus left, Azure.ResourceManager.Sphere.Models.CertificateStatus right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ClaimDevicesContent
+    {
+        public ClaimDevicesContent(System.Collections.Generic.IEnumerable<string> deviceIdentifiers) { }
+        public System.Collections.Generic.IList<string> DeviceIdentifiers { get { throw null; } }
+    }
+    public partial class CountDeviceResponse : Azure.ResourceManager.Sphere.Models.CountElementsResponse
+    {
+        internal CountDeviceResponse() { }
+    }
+    public partial class CountElementsResponse
+    {
+        internal CountElementsResponse() { }
+        public int Value { get { throw null; } }
+    }
+    public partial class DeviceGroupPatch
+    {
+        public DeviceGroupPatch() { }
+        public Azure.ResourceManager.Sphere.Models.AllowCrashDumpCollection? AllowCrashDumpsCollection { get { throw null; } set { } }
+        public string Description { get { throw null; } set { } }
+        public Azure.ResourceManager.Sphere.Models.OSFeedType? OSFeedType { get { throw null; } set { } }
+        public Azure.ResourceManager.Sphere.Models.RegionalDataBoundary? RegionalDataBoundary { get { throw null; } set { } }
+        public Azure.ResourceManager.Sphere.Models.UpdatePolicy? UpdatePolicy { get { throw null; } set { } }
+    }
+    public partial class DeviceInsight
```

Comment:
Could we rename this type to `SphereDeviceInsight`?

Reply:
fixed in latest commit